From a3fe40da4b0ce74bc32263737b615c087a5050a7 Mon Sep 17 00:00:00 2001
From: Siddhesh Poyarekar <siddhesh.poyarekar@gmail.com>
Date: Fri, 21 Jan 2011 14:11:32 +0530
Subject: [PATCH] SourceDelegate also should have a SORTPRIORITY

Children of SourceDelegate (PanelDelegate) are also put into the
details tree view, so it should also have a SORTPRIORITY.
---
 lib/userprofile.py |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/lib/userprofile.py b/lib/userprofile.py
index 4308874..44a93f7 100755
--- a/lib/userprofile.py
+++ b/lib/userprofile.py
@@ -164,6 +164,8 @@ class SourceDelegate:
         self.name = delegate_name
         self.source = source
         self.namespace_section = namespace_section
+        # Set the priority level for sorting in Details view, lower numbers appear first
+        self.SORTPRIORITY = 1000
 
     def get_name (self):
         """Returns the configuration delegate's name."""
-- 
1.7.3.4

